docs/proxyserver/proxyserver.md

```diff
@@ -37,8 +38,8 @@ data:
 * `path`: The path for agents to send requests.
 * `sub-resource`: The resource of requests from agents.
 * `use-id`: If true, the path of requests includes cluster name as ID. If false, there is no ID in the path.
-* `secret`: The secret with CA information to access the backend service. Format is `namespace/<secret-name>`.
-
+* `secret`: The secret with signed serving certificate and key pair information to access the backend service. Format is `namespace/<secret-name>`.
+* `caConfigMap`: The configmap with CA certificate information to access the backend service. Format is `namespace/<configMap-name>`.
```

Comment:
Is `namespace` is optional? The search-chart PR didn't have a namespace.

Reply:
Yes, if it is not there, the proxy configmap's namespace is used.